src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/FilterUIHintAttribute.cs

```diff
@@ -62,7 +62,7 @@ public FilterUIHintAttribute(string filterUIHint, string presentationLayer)
         /// <param name="filterUIHint">The name of the control</param>
         /// <param name="presentationLayer">The presentation layer</param>
         /// <param name="controlParameters">The list of parameters for the control</param>
-        public FilterUIHintAttribute(string filterUIHint, string presentationLayer,
+        public FilterUIHintAttribute(string filterUIHint, string? presentationLayer,
```

Comment:
should controlParameters be `object?[]`?

Reply:
Thanks!